### PR TITLE
Lazy loading user balances on swap page using rpc calls

### PR DESCRIPTION
**Pull request by pawan-tezsure**

### CODE REVIEW

File: src/apis/swap/swap.js [hunk 1]
```diff
@@ -775,6 +775,66 @@ export const computeRemoveTokens = (
...
+    console.log(url);
+    const response = await axios.get(url);
+    if (mapId === 3956 || mapId === 4353) {
+      balance = response.data.args[0].args[1].int;
+    } else if (mapId === 3943) {
+      balance = response.data.args[1].int;
+    } else if (mapId === 199 || mapId === 36 || mapId === 6901) {
+      balance = response.data.args[0].int;
+    } else if (
+      mapId === 1777 ||
```
Comment (vijaypushkin) [inline src/apis/swap/swap.js:797]: extract all these numbers and assign it to a variable with proper name, so developers after you can understand what it does

```js
const dataWithSecondLevelArg = [3956, 4353]

// or

/**
 * balance for these mapIds will be present in `response.data.args[0].args[1].int`
*/
const type2Data = [3956, 4353]

// then use it like this

if (dataWithSecondLevelArg.includes(mapId) {
  // noop
}

**Review by vijaypushkin — CHANGES_REQUESTED** [review]

File: src/apis/swap/swap.js (hunk 1 above)
Comment (vijaypushkin) [reply src/apis/swap/swap.js:797]: Also, remove unwanted codes,

Comment (pawan-tezsure) [reply src/apis/swap/swap.js]: Done

File: src/apis/swap/swap.js [hunk 2]
```diff
@@ -775,6 +775,66 @@ export const computeRemoveTokens = (
...
+    /**
+     * balance for these type1MapIds will be present in `response.data.args[0].args[1].int`
+     * balance for these type2MapIds will be present in `response.data.args[1].int`
+     * balance for these type3MapIds will be present in `response.data.args[0].int`
+     * balance for these type4MapIds will be present in `response.data.int`
+     * balance for these type5MapIds will be present in `response.data.args[0][0].args[1].int`
+     * Else condition basically focuses for lpTokens since they are large in numbers and all same
+     */
+    const type1MapIds = [3956, 4353];
+    const type2MapIds = [3943];
```
Comment (vijaypushkin) [inline src/apis/swap/swap.js:799]: Add respective JSDoc to each of these variable, not all to the first one

**Review by vijaypushkin — CHANGES_REQUESTED** [review]

File: src/apis/swap/swap.js (hunk 2 above)
Comment (pawan-tezsure) [reply src/apis/swap/swap.js:809]: Done

**Review by vijaypushkin — APPROVED** [review]

**Pull request merged**